packages/shipjs/src/step/release/createGitTag.js

```diff
@@ -5,7 +5,9 @@ export default ({ version, config, dir, dryRun }) =>
   runStep({ title: 'Creating a git tag.' }, () => {
     const { getTagName } = config;
     const tagNames = arrayify(getTagName({ version }));
-    const command = tagNames.map((tag) => `git tag ${tag}`).join(' && ');
+    const command = tagNames
+      .map((tag) => `git tag -a ${tag} -m ${tag}`)
```

Comment:
Won't this cause an issue for people relying on shipjs, as it changed from plain to annotated tag? Should this be a breaking or an option?

Reply:
An annotated tag shows up in a couple more places than a regular tag (git describe for example), but I don't see how that behaviour would be intentional for anyone, rather just a fix :)